packages/vscode-extension/src/treeview/treeViewManager.ts

```diff
@@ -49,6 +49,15 @@ class TreeViewManager {
         undefined,
         { name: "new-folder", custom: false }
       ),
+      new TreeViewCommand(
+        StringResources.vsc.commandsTreeViewProvider.initProjectTitleNew,
+        StringResources.vsc.commandsTreeViewProvider.initProjectDescription,
+        "fx-extension.init",
+        vscode.TreeItemCollapsibleState.None,
+        undefined,
+        undefined,
+        { name: "new-folder", custom: false }
```

Comment:
Use the same folder with 'create project'?

Reply:
this is the draft UX and will be continuously improved according to PM's design.